Makefile

```diff
@@ -232,6 +232,23 @@ docker-run:
 	@echo running wp-calypso docker image at localhost:3000
 	docker run -it --name wp-calypso --rm -p 80:3000 -e NODE_ENV='wpcalypso' -e CALYPSO_ENV='wpcalypso' wp-calypso
 
+# Desktop specific tasks
+# These are just convenience commands that run tasks from desktop/Makefile
+desktop-run:
+	@cd desktop; make run
+
+desktop-osx:
+	@cd desktop; make osx
+
+desktop-linux:
+	@cd desktop; make linux
+
+desktop-win32:
+	@cd desktop; make win32
+
+desktop-test:
+	@cd desktop; make test
```

Comment:
🤔 That might be crazy enough to work 😄

Calling this from root:
```make
subshell-test:
  @pwd
  @cd desktop; pwd;
  @pwd
  @cd desktop; pwd;
```
Outputs:
```cli
.../automattic/calypso
.../automattic/calypso/desktop
.../automattic/calypso
.../automattic/calypso/client
```
And leaves the calling shell in the root as expected :)